.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: CI
+
+on:
+  workflow_run:
+    workflows: [image]
+    types:
+      - completed
+    branches:
+      - main
+
+jobs:
+  e2e-tests-nvidiadriver:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        driver:
+          - 535.183.06
+          - 550.90.07
```

Comment:
An idea for a potential follow-up -- instead of defining a matrix and spinning up one AWS instance per driver version, can we instead pass all driver versions as input to the test script and test all of them in sequence? For example, first install 535.183.06, and then upgrade to 550.90.07? This would also allow us to source the driver versions from the `versions.mk` file, instead of having to redefine and maintain that list here.